TheWarWithin/RogueSubtlety.lua

```diff
@@ -876,9 +879,8 @@ spec:RegisterAbilities( {
         notalent = "gloomblade",
 
         cp_gain = function ()
-            if buff.shadow_blades.up then return 7 end
-            if buff.premeditation.up then return combo_points.max end
-            return 1 + ( buff.broadside.up and 1 or 0 )
+            if buff.shadow_blades.up or buff.premeditation.up then return combo_points.max
```

Comment:
Avoid unnecessary `else`s.  Since the first condition results in a return you can `end` there and leave `return 1` to stand alone.

```
if buff.shadow_blades.up or buff.premeditation.up then return combo_points.max end
return 1
```

Reply:
good point, will attempt to store that in the brain longterm.